File: JavaConverter/src/Conversor/ClassesGeradas/Carro.js
var Carro = (function (){
      function Carro(){
           if(this.nome === undefined)
                this.nome = null; 
           if(this.ano === undefined)
                this.ano = null; 
           if(this.modelo === undefined)
                this.modelo = null; 
           if(this.marca === undefined)
                this.marca = null; 
           if(this.combustivel === undefined)
                this.combustivel = null; 
           if(this.valor === undefined)
                this.valor = null; 
      }

      Carro.prototype.getnome = function() {
          return this.nome;
      };
      Carro.prototype.setnome(nome)= function(nome) {
          this.nome = nome;
      }; 
      Carro.prototype.getano= function() {
          return this.ano;
      };
      Carro.prototype.setano(ano)= function(ano) {
          this.ano = ano;
      }; 
      Carro.prototype.getmodelo= function() {
          return this.modelo;
      };
      Carro.prototype.setmodelo(modelo)= function(modelo) {
          this.modelo = modelo;
      }; 
      Carro.prototype.getmarca= function() {
          return this.marca;
      };
      Carro.prototype.setmarca(marca)= function(marca) {
          this.marca = marca;
      }; 
      Carro.prototype.getcombustivel= function() {
          return this.combustivel;
      };
      Carro.prototype.setcombustivel(combustivel)= function(combustivel) {
          this.combustivel = combustivel;
      }; 
      Carro.prototype.getvalor= function() {
          return this.valor;
      };
      Carro.prototype.setvalor(valor)= function(valor) {
          this.valor = valor;
      }; 


Carro.prototype.toString = function() {
return "Carro: {"+"nome= "+this.nome+",ano= "+this.ano+",modelo= "+this.modelo+",marca= "+this.marca+",combustivel= "+this.combustivel+",valor= "+this.valor+"}";
};
return Carro;
});

Carro["__class"] = "Carro";

File: JavaConverter/src/Conversor/ClassesGeradas/Carro.cpp
#include <iostream>  
using namespace std;

class Carro{ 

public: 
      string nome; 
      int ano; 
      string modelo; 
      string marca; 
      string combustivel; 
      double valor; 

public: 
      string getnome() 
      {
             return nome;
      } 

      int getano() 
      {
             return ano;
      } 

      string getmodelo() 
      {
             return modelo;
      } 

      string getmarca() 
      {
             return marca;
      } 

      string getcombustivel() 
      {
             return combustivel;
      } 

      double getvalor() 
      {
             return valor;
      } 

      void setnome(String nome) 
      {
             this->nome = nome;
      } 

      void setano(int ano) 
      {
             this->ano = ano;
      } 

      void setmodelo(String modelo) 
      {
             this->modelo = modelo;
      } 

      void setmarca(String marca) 
      {
             this->marca = marca;
      } 

      void setcombustivel(String combustivel) 
      {
             this->combustivel = combustivel;
      } 

      void setvalor(double valor) 
      {
             this->valor = valor;
      } 



string Carro ::toString() {
      std::string texto = "Carro{"+"nome:" + std::to_string( this->getnome())
      +"nome:" + std::to_string( this->getnome())
      +"ano:" + std::to_string( this->getano())
      +"modelo:" + std::to_string( this->getmodelo())
      +"marca:" + std::to_string( this->getmarca())
      +"combustivel:" + std::to_string( this->getcombustivel())
      +"valor:" + std::to_string( this->getvalor())+"}"
}

